alacritty/src/logging.rs

```diff
@@ -108,7 +108,29 @@ impl log::Log for Logger {
         }
 
         let now = time::strftime("%F %T.%f", &time::now()).unwrap();
-        let msg = format!("[{}] [{:<5}] [{}] {}\n", now, record.level(), target, record.args());
+
```

Comment:
The stuff in logging.rs is getting kinda long, do you think it would be worth it to move the creation of the log text into a separate function? There shouldn't be too many arguments (only record?) and the output is a clean `String`.